utils/tracing/win/Start-Tracing.ps1

```diff
@@ -0,0 +1,28 @@
+param(
+   [bool]$IncludeGeneralTrace = $False
+)
+

```

Comment:
Can you use the `begin`, `process`, `end` paradigm? That might even remove the need for a separate `Stop-Tracing.ps1`